src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -119,6 +127,15 @@ public final void setExcludedClasses(String... excludedClasses) {
             Collections.unmodifiableSet(Arrays.stream(excludedClasses).collect(Collectors.toSet()));
     }
 
+    /**
+     * Sets user-excluded regular expression of classes to ignore.
+     * @param excludeClassesRegexp the string representing regular expression of classes to ignore.
+     */
+    public void setExcludeClassesRegexp(String excludeClassesRegexp) {
```

Comment:
Method parameter should be of type `Pattern`. Utility will do conversion for you.